src/Umbraco.Web/Editors/MomentController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Web.Http;
+using Umbraco.Core.IO;
+using Umbraco.Web.Mvc;
+
+namespace Umbraco.Web.Editors
+{
+    [PluginController("UmbracoApi")]
+    public class MomentController : UmbracoAuthorizedJsonController
```

Comment:
Consider turning this controller into something that could be used for more generic purposes in the future. Something along the lines of `BackOfficeAssetsController` and then change the method to `GetSupportedMomentLocales`, this would allow us to re-use this controller for any future asset requirements.

Reply:
Great idea!